src/Maestro/Microsoft.DotNet.Maestro.Tasks/src/Models/Manifest.cs

```diff
@@ -30,6 +30,34 @@ public class Manifest
 
         [XmlAttribute(AttributeName = "Location")]
         public string Location { get; set; }
+
+        #region Properties to be used in new publishing flow
+
+        [XmlAttribute(AttributeName = "InitialAssetsLocation")]
+        public string InitialAssetsLocation { get; set; }
+
+        [XmlAttribute(AttributeName = "AzureDevOpsBuildId")]
+        public int? AzureDevOpsBuildId { get; set; }
+
+        [XmlAttribute(AttributeName = "AzureDevOpsBuildDefinitionId")]
+        public int? AzureDevOpsBuildDefinitionId { get; set; }
+
+        [XmlAttribute(AttributeName = "AzureDevOpsAccount")]
+        public string AzureDevOpsAccount { get; set; }
+
+        [XmlAttribute(AttributeName = "AzureDevOpsProject")]
+        public string AzureDevOpsProject { get; set; }
+
+        [XmlAttribute(AttributeName = "AzureDevOpsBuildNumber")]
+        public string AzureDevOpsBuildNumber { get; set; }
```

Comment:
Isn't this the same as BuildId? I think that's what BAR returns as the build number right now.  It's the build id in at least all cases except for aspnet, who has their overall version number.  What is it used for?

Reply:
Ahh I think I see now? There are now both Github and AzDO versions of these properties, but Branch and Build and such were not removed from the manifest?

Reply:
My original thinking was that this updated Manifest could still be used to submit tasks to the old (current) version of the Maestro API. That's why I didn't remove the branch, buildid, location properties. But as soon as everything is using the new api version we should be good to remove them.

Reply:
I don't use that in the publishing stuff. This field will be used eventually on the new MC.

Reply:
Yes, that's what BAR currently stores as BuildNumber. The new API handles the conversion between old BuildNumber <--> AzureDevOpsBuildNumber